gui/src/home/home-browser-window.js

```diff
@@ -0,0 +1,48 @@
+
+const { BrowserWindow } = require('electron')
+const { ApkDependencyController } = require('./apk-dependency-analyzer-controller')
+
+const HOME_WINDOW_WIDTH  = 360
```

Comment:
just curious: how have you defined these numbers?

Reply:
I have used these numbers because I find it better for window size, just for the home page because it is small... basically, I have tested some values and these values are the best I found for this.